ee/clickhouse/queries/util.py

```diff
@@ -41,7 +42,7 @@ def format_ch_timestamp(timestamp: datetime, filter, default_hour_min: str = " 0
         or (filter._date_from == "-24h")
         or (filter._date_from == "-48h")
     )
-    return timestamp.strftime("%Y-%m-%d{}".format(" %H:%M:%S.%f" if is_hour_or_min else default_hour_min))
+    return timestamp.strftime("%Y-%m-%d{}".format(" %H:%M:%S" if is_hour_or_min else default_hour_min))
```

Comment:
@timgl was there a reason we needed microseconds here?

Reply:
Yes! we use this to do ?after= on events list. If it's only precise to seconds we might miss some events in that list.

Reply:
ah i see let me work that back in

Reply:
oh the event conditions for event list are separate so this doesn't affect it which is why no tests failed

Reply:
Ah great, in that case go ahead